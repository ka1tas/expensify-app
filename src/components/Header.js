import React from 'react';
import {NavLink} from "react-router-dom";

    const Header = ()=>(
    <div>
        <header>
            <h1>Expensify.</h1>
            <div>
                <h3>
                <span>
                    <NavLink to={'/'} activeClassName={'is-active'} exact={true}>Home  &nbsp;&nbsp; </NavLink>
                    <NavLink to={'/create'} activeClassName={'is-active'}>Add Expense   &nbsp;&nbsp;</NavLink>
                    <NavLink to={'/help'} activeClassName={'is-active'}>Help</NavLink>

                </span>
                </h3>
            </div>
        </header>
         <hr />
        </div>
    );

    export default Header;
