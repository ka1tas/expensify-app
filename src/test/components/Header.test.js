import React from 'react';
import { shallow } from "enzyme";
import Header from '../../components/Header';

test('Should render header',()=> {

    const wraper = shallow(<Header/>);
    //  expect(wraper.find('h1').length).toBe(1);
    // expect(wraper.find('h1').text()).toEqual('Expensify.');
    expect(wraper).toMatchSnapshot();


})


