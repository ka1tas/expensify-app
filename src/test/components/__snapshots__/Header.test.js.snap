// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render header 1`] = `
<div>
  <header>
    <h1>
      Expensify.
    </h1>
    <div>
      <h3>
        <span>
          <NavLink
            activeClassName="is-active"
            exact={true}
            to="/"
          >
            Home     
          </NavLink>
          <NavLink
            activeClassName="is-active"
            to="/create"
          >
            Add Expense     
          </NavLink>
          <NavLink
            activeClassName="is-active"
            to="/help"
          >
            Help
          </NavLink>
        </span>
      </h3>
    </div>
  </header>
  <hr />
</div>
`;
